ocaml/xapi/xapi_pool_update.ml

```diff
@@ -259,6 +260,13 @@ let parse_update_info xml =
       with
       | _ -> []
     in
+    let enforce_homogeneity =
+      try
+        List.assoc "enforce-homogeneity" attr
+        |> fun s -> s = "1" || String.lowercase s = "true"
```

Comment:
We are currently not on 4.03.0 (I believe) and this can't use `String.lowercase_ascii`.